CHANGES.md

```diff
@@ -69,6 +69,8 @@
 
 - Fix a crash when formatting some dicts with parenthesis-wrapped long string keys
   (#3262)
+- Reformat empty and whitespace-only files as either an empty file (if no newline is
```

Comment:
This should be moved up.

Reply:
You mean as the first entry of the "Preview style" section?

Reply:
This is under the 22.10.0 section, which isn't right. Please move it (up) to the unreleased section :)